cabal.project

```diff
@@ -139,7 +139,8 @@ package small-steps-test
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/Win32-network
-  tag: d81187239fe5f303aa1770565ad307efa605c820
+  tag: f71be820aaa480256bba24fc3f2a1e12cd1eab0f
+  --sha256: 1ggm3239z41z0ca9al43v9y2wmf20bbmdgj1k27kyph54gqkdbhn
```

Comment:
Use: `94153b676617f8f33abe8d8182c37377d2784bd1` tag, `f71be` is not on `master` anymore.